Dockerfile

```diff
@@ -2,20 +2,22 @@ FROM --platform=linux/amd64 rocker/tidyverse
 
 # install system dependencies for R packages
 RUN apt-get update \
```

Comment:
nit: this seems unnecessary now that you're not installing `git`, but I suppose you left it in to make it easy to add system installs in the future?

Reply:
Correct, all of this is just the skeleton/ infrastructure for a "very soon-to-come" `main.R` script, that will likely have at least SOME system dependencies.